src/main/java/io/strimzi/test/container/StrimziKafkaCluster.java

```diff
@@ -412,6 +415,50 @@ public void start() {
                         return false;
                     }
                 });
+        } else if (this.isKraftKafkaCluster()) {
+            // Readiness check for KRaft mode
+            Utils.waitFor("Kafka brokers to form a quorum", Duration.ofSeconds(5).toMillis(), Duration.ofMinutes(1).toMillis(),
```

Comment:
isn't 5 sec too much for pooling?

Reply:
Good catch ... we probably want it to be fairly aggressive.

Reply:
Well, I can try 2 seconds. WDYT @Frawless? The tests are no longer flaky with this change
```java
Tests run: 177, Failures: 0, Errors: 0
```

Reply:
This is with 5s polling so I will try 1s and see how time execution will change.
```
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  21:22 min
[INFO] Finished at: 2024-10-16T12:28:53Z
```

Reply:
I would go to 1 second I guess.

Reply:
Or make it configurable 😉

Reply:
```
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  20:05 min
[INFO] Finished at: 2024-10-16T13:13:17Z
```
For 1s :) That's a solid improvement.